src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,76 @@
 package core.basesyntax;
 
 public class SalaryInfo {
+    public static final int ZERO = 0;
+    public static final int FIRST = 1;
+    public static final int SECOND = 2;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        final String DivideR = "[.]";
```

Comment:
refactor naming everywhere

Reply:
I can`t, cause when I write all constants from capital letters, I have the next message in maven: 
 Abbreviation in name 'DIVIDER' must contain no more than '2' consecutive capital letters. [AbbreviationAsWordInName]

Reply:
instead of this, I created constants variable in the class field and made them "public static"